content-repo/extra-docs/releases/20.12.1.md

```diff
@@ -0,0 +1,1284 @@
+
+#### Layouts
+- **Qradar Generic**
+- Added usernames field to layout.
+
+---
+
+### Indeni Pack v1.0.7 (Partner Supported)
+#### Incident Fields
+- **Indeni Issue ID**
+- **Indeni Device ID**
+
+---
+
+### Infinipoint Pack v1.0.3 (Partner Supported)
+#### Integrations
+##### Infinipoint
+- Maintenance and stability enhancements.
+
+---
+
+### JSON Feed Pack v1.1.0
+#### Integrations
+
+---
+
+### Lastline Pack v1.0.3
+#### Integrations
+##### Lastline v2
+- Added support for account-based authentication.
+- Upgraded the Docker image to demisto/python3:3.8.6.13358.
+
+---
+
+### LogRhythmRest Pack v1.0.2
+#### Integrations
+##### LogRhythmRest
+Improved implementation of error handling in case an unexpected content type was returned.
+
+---
+
+### Logz.io Pack v1.1.3 (Partner Supported)
+#### Integrations
+##### Logz.io
+Fixed integration argument bug when fetching event logs from Logz.io.
+
+---
+
+### MITRE ATT&CK Pack v1.1.7
+#### Indicator Fields
+##### MITRE System Requirements
+- **MITRE Platforms**
+- **MITRE Extended Aliases**
+- **MITRE Description**
+- **MITRE Contributors**
+- **MITRE Aliases**
+- **MITRE Labels**
+- **MITRE ID**
+- **MITRE Permissions Required**
+- **MITRE Kill Chain Phases**
+- **MITRE Detection**
+- **MITRE Type**
+- **MITRE Version**
+- **MITRE Impact Type**
+- **MITRE Data Sources**
+- **MITRE External References**
+- **MITRE Name**
+- **MITRE Defense Bypassed**
+
+---
+
+### Majestic Million Feed Pack v1.0.1
+#### Integrations
+##### Majestic Million Feed
+Maintenance and stability enhancements.
+
+---
+
+### Microsoft Cloud App Security Pack v1.0.12
+#### Incident Types
+
+---
+
+### Microsoft Defender Advanced Threat Protection Pack v1.2.6
+#### Integrations
+##### Microsoft Defender Advanced Threat Protection
+- Added the **timeout** argument to the ***microsoft-atp-advanced-hunting*** command.
+- Upgraded the Docker image to demisto/crypto:1.0.0.14297.
+
+---
+
+### NIST Pack v1.0.5
+#### Incident Fields
+- **Exactly what happened, and at what times?**
+- **What information was needed sooner?**
+- **Were any steps or actions taken that might have inhibited the recovery?**
+- **How could information sharing with other organizations have been improved?**
+- **What additional tools or resources are needed to detect, analyze, and mitigate future incidents?**
+- **What corrective actions can prevent similar incidents in the future?**
+- **How well did staff and management perform in dealing with the incident? Were the documented procedures followed? Were they adequate?**
+- **What precursors or indicators should be watched for in the future to detect similar incidents?**
+- **What would the staff and management do differently the next time a similar incident occurs?**
+
+---
+
+### NTT Cyber Threat Sensor Pack v1.0.2 (Partner Supported)
+#### Incident Fields
+- **Graph Plot**
+- **FAERE Description**
+
+---
+
+### PAN-OS Pack v1.6.9
+#### Incident Fields
+
```

Comment:
- **Target Firewall Version** 
Updated the entity to conform to the new schema standards.